lib/recommendation-server.js

```diff
@@ -27,7 +27,7 @@ function RecommendationServer(opts) {
   this.timeout = opts.timeout;
   // Note: this is the xhr service, not an xhr instance
   this.xhr = opts.xhr;
-  this.console = opts.console;
+  this.logger = new opts.Logger('server', opts.win);
```

Comment:
because the logger contains a window pointer, it's probably best to `delete this.logger` in all the `destroy` methods